src/components/implementation/no_interface/new_boot/boot_deps.h

```diff
@@ -28,24 +38,24 @@ boot_comp_map_memory(struct cobj_header *h, spdid_t spdid, pgtblcap_t pt)
 	int flag;
 	vaddr_t dest_daddr, prev_map = 0;
 	int tot = 0, n_pte = 1;	
-	
 	struct cobj_sect *sect = cobj_sect_get(h, 0);
 	
-	/*Expand Page table, could do this faster*/
-	for(j = 0; j < (int)h->nsect; j++){
+	/* Expand Page table, could do this faster */
```

Comment:
Doesn't the previous boot structure (e.g. for `llbooter`) *not* put this whole function in boot_deps?  Doesn't it just call to boot_deps for the actual mapping, and not for the cobj parsing?